tests/unit/ds-macro-test.js

```diff
@@ -37,3 +37,32 @@ test('DS macro', function(assert) {
 
   assert.deepEqual(relationships, ['user', 'car']);
 });
+
+class ES6Model extends DS.Model {
```

Comment:
You should probably define the class inside the body of the test itself, see the `@action` decorator test for an example (sorry for the spam, commenting on mobile and kept putting it in the wrong place 😛)